genie/dashboardTemplate.Rmd

```diff
@@ -525,4 +525,41 @@ if (!is.null(this_bed)) {
   kable(final_matrix,
         caption = "Distribution of top 5 most frequently mutated genes per SEQ_ASSAY_ID")
 }
+```
+
+
+Each panel's bed file is used to generate a gene panel file used in cBioPortal. All bed files undergo gene symbol remapping during processing.
+
+* `num_genes_remapped`: number of unique genes a panel has after submitted genes are remapped
```

Comment:
Suggested changes to explanatory text:
num_genes_remapped: number of unique genes a panel has after submitted genes are remapped
num_genes_submitted: number of unique genes found in site's submitted BED file prior to remapping
num_genes_expected: size of panel as defined in site's submitted assay_information.txt file